src/client/Licit.js

```diff
@@ -615,6 +618,26 @@ class Licit extends React.Component<any, any> {
     view.dispatch(tr.setSelection(TextSelection.atEnd(view.state.doc)).scrollIntoView());
     view.focus();
   };
+  //To insert empty text property for content without text attribute
+  insertTextNodes = (docJson) => {
+    // Create a deep copy of docJson to avoid modifying the original object
+    const deepCopy = JSON.parse(JSON.stringify(docJson));
+
+    deepCopy.content.forEach((node) => {
+        if (node.content) {
+            node.content = node.content.map((content) => {
+                if (content.type === 'text' && !Object.hasOwn(content, 'text')) {
```

Comment:
`!!content.text` would suffice here. hasOwn may be true here, but the value still set to null or undefined.

Reply:
I have some doubt here, we are looking for text nodes and it should be without 'text' property. So just checking for a text property won't be enough right ? You can have other nodes like 'table-cell' without 'text' property and we don't have to add in that case